README.md

```diff
@@ -790,6 +790,24 @@ May also be configured in `lerna.json`:
 }
 ```
 
+#### --npm-client-args [client]
```

Comment:
Do we want to isolate it to `bootstrap` command only? I thought it should be for all of them regardless of where npm client is used.